packages/dart/build.sh

```diff
@@ -32,6 +32,12 @@ termux_step_get_source() {
 	gclient sync -D --force --reset
 }
 
+termux_step_pre_configure() {
+	sed -i -e 's:\([^A-Za-z0-9_]\)/usr/bin:\1'$TERMUX_PREFIX'/local/bin:g' \
```

Comment:
I think this should be `/system/bin`, not `$PREFIX/local/bin`, as I don't think any packages install to the latter.

@Grimler91, you added the patch he's replacing, what do you think?

Reply:
User could install custom tar version to $PREFIX/local/bin for some reason, so checking there makes some sense.

Falling back to /system/bin/tar could work as well, fallback should in any case not bee needed since dart depend on tar.

@xtkoba, any particular reason for sed'ing instead of having a patch? I think a patch is easier to maintain, if the path `/usr/bin` would be moved to another file for example then sed here will just silently not do anything and after a package upgrade we risk ending up with a package where the tar-path is broken

Reply:
I do not have access to the file to be patched.